installation/vs-code-integration/new-project-wizard.md

```diff
@@ -28,7 +28,7 @@ You can start the Telerik UI for .NET MAUI Template Wizard in either of the foll
     1. Select **MAUI** tab
     1. Click **Create New Project**.
     1. The **Telerik UI for .NET MAUI Template Wizard** opens
-    1. Using the Template Wizard you can create a new **Blank App** or **Blank Shell App**.
+    1. Using the Template Wizard you can create a new **Blank App** or **Blank Shell App**. You can create a project with a predefined theme using the **USE TELERIK THEMING** option.
```

Comment:
I suggest placing this information in another section dedicated to the project configuration (the `#configure-the-project` section). Selecting the theming option isn't a step in the procedure described here (starting the wizard).
```suggestion
    1. Using the Template Wizard you can create a new **Blank App** or **Blank Shell App**.
```